elyra/pipeline/validate.py

```diff
@@ -277,8 +277,9 @@ async def _validate_node_properties(self, pipeline: dict, pipeline_runtime: str,
                                                                   response=response)
 
                         # Check label against kfp naming standards
-                        if pipeline_runtime == 'kfp' and node_label and filename != node_label:
-                            self._validate_label(node_id=node['id'], node_label=node_label, response=response)
+                        if pipeline_runtime == 'kfp' and node_label:
```

Comment:
Will update this, I think there was a particular version of the pipeline (2.0.x or 2.1.x?) where I was encountering empty labels. This problem probably doesnt exist anymore since it will all go through the migration process and fields are updated. But just as another precaution will move the falsey check into the label validation.